binding/web/src/cobra.ts

```diff
@@ -286,19 +327,19 @@ export class Cobra implements CobraEngine {
     }
 
     return {
-      frameLength: frameLength,
-      inputBufferAddress: inputBufferAddress,
+      malloc: exports.malloc,
```

Comment:
is there a reason we put malloc here?

Reply:
I was just copying Cheetah's exports - does Cheetah need it? What is it used for?

Reply:
doesnt need it, I removed it

Reply:
cool I'll do the same